### ISSUE TITLE
Staging Review finding: Error messages are not announced when interacting with file upload

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#125965]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Form Upload
**Product:** Form Upload
**Feature:** Multi-file upload (Adding supporting evidence)

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** Yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When there is an error message in file upload and the user navigates back to the file upload and focuses on "Change file" or "Delete", there's no indication that there is an error state. A screen reader user is left guessing at the current state of the file upload, and may not know how to proceed. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="623" height="361" alt="Forbidden error" src="https://github.com/user-attachments/assets/d14013f6-f942-444b-b15d-524b80183e1e" />

</details>

## Recommended action

Ensure that the error is always discoverable through expected means. In this case, associating the error to the group so that it is read out on focus within the group is a potential fix to ensure that screen reader users are aware of the issue. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** N/A
- **Design System Component:** [Form - File input component](https://design.va.gov/components/form/file-input)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jeana-adhoc** [comment]
Carrying over - can't test/close this until component is released
